### PR TITLE
API creation with scraper and OCR implemented

### PR DESCRIPTION
**Pull request by noahnisbet**
changes for deploying scraper/ocr in a fastapi app (commit history on https://github.com/noahnisbet/human-rights-first-asylum-ds-noahnisbet)

### CODE REVIEW

**Review by codealamode — APPROVED** [review]
Looks good

**Comment by rdukewiesenb** [comment]
Architecture is clean and easy to build upon. Transferring scraper from .ipynb to .py file makes code functional and able to be implemented.

**Comment by rdukewiesenb** [comment]
Architecture is clean and easy to build upon. Transferring scraper from .ipynb to .py file makes code functional and able to be implemented.

File: Aptfile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+poppler-utils
```
Comment (rdukewiesenb) [inline Aptfile:1]: This is a great package for NLP.

File: app/ocr.py [hunk 2]
```diff
@@ -7,64 +8,115 @@
...
-    # query = """INSERT INTO pdfs(pdf, plaintext) VALUES (%s, %s)""" 
-    # vals = (file, text)
-    # engine.execute(query, vals)
-    # Return text as response
-    return {'Text': text}
+    
+    text = []
 
+    ### Converts the bytes object recieved from fastapi
+    pages = convert_from_bytes(file,200,fmt='png',thread_count=2)
```
Comment (rdukewiesenb) [inline app/ocr.py:40]: Great way to scrape text PDFs that present more like an image.

File: app/ocr.py [hunk 3]
```diff
@@ -7,64 +8,115 @@
...
+    ### Getting references / sex of applicant
+    references = [
+    'Matter of AB, 27 I&N Dec. 316 (A.G. 2018)' 
+    if case.references_AB27_216() else None,
+    'Matter of L-E-A-, 27 I&N Dec. 581 (A.G. 2019)'
+    if case.references_LEA27_581() else None
+    ]
+
+    case_data['references'] = '; '.join([r for r in references if r])
+    case_data['sex_of_applicant'] = case.get_seeker_sex()
```
Comment (rdukewiesenb) [inline app/ocr.py:119]: I will be adding further fields through my pull requests.

**Review by rdukewiesenb — APPROVED** [review]

**Pull request merged**